receiver/libhoneyreceiver/go.mod

```diff
@@ -68,7 +68,7 @@ require (
 	go.opentelemetry.io/otel/metric v1.32.0 // indirect
 	go.opentelemetry.io/otel/sdk v1.32.0 // indirect
 	go.opentelemetry.io/otel/sdk/metric v1.32.0 // indirect
-	go.opentelemetry.io/otel/trace v1.32.0 // indirect
```

Comment:
this doesn't look right... either the indirect needs to stay or the import has to be moved up into the other group